HaltonHills.md

```diff
@@ -0,0 +1,11 @@
+# Halton Hills, Ontario
+
+### Where to eat
+- John's Fries (Chip Truck)
+- Tim Hortons
+
+### What to do
+- Play softball at the Ball Diamond.
+- Skateboard at the Church
+- Visit the Scrapyard
+- Play Golf at the Country Club
```

Comment:
@futurepr0n Lots of outdoors activities! Sounds like the weather is pretty good. 😄 ☀️